packages/password-input/src/PasswordInput/PasswordInput.tsx

```diff
@@ -171,4 +169,4 @@ PasswordInput.propTypes = {
   size: PropTypes.oneOf(Object.values(Size)),
   value: PropTypes.string,
   stateNotifications: stateNotificationCheck,
-} as unknown;
+} as any;
```

Comment:
When using `unknown` i kept getting the error 

```Type 'unknown' is not assignable to type 'WeakValidationMap<PasswordInputProps & RefAttributes<HTMLInputElement>> | undefined'.ts(2322)```

but I added `ts-ignore` above to ignore it, but switching to `any` removes that warning. I was using `unknown` before to stop interfering with TS types but I think i could also get away with that using `any`. @TheSonOfThomp @spark33 thoughts?

Reply:
Ah yeah, I think `as unknown` only works for regular components, not `forwardRef`. This should work for now